drivers/jason0x43-wemo_switch.groovy

```diff
@@ -179,3 +188,84 @@ private debugLog(message) {
 private syncTime() {
     parent.childSyncTime(device)
 }
+
+
+def updatePort(port) {
+    iPort=s2i(port)
+    if (iPort < 1 || iPort > 65535 ) {
+        infoLog("Invalid TCP port specified - ${port}")
+        -1
+    } else {
+        hPort=i2h(iPort)
+        def existingPort = device.getDataValue('port')
+
+        infoLog("Existing Port: ${h2i(existingPort)}")
+        
+        if (iPort && iPort != h2i(existingPort)) {
+            debugLog("Updating port from ${h2i(existingPort)} to ${port}")
+            device.updateDataValue('port', hPort)
+        }
+        
+    }
+}
+
+def updateAddress(ip) {
+    
```

Comment:
The code for this function should go in the parent app so that all drivers can use it.